api/v1/openapi.yaml

```diff
@@ -960,6 +975,13 @@ definitions:
         "$ref": "#/definitions/Labels"
       to:
         "$ref": "#/definitions/Labels"
+      dports:
+        description: |
+          List of Layer 4 port and protocol pairs which the identity will use for
```

Comment:
>List of Layer 4 port and protocol pairs which will be used in communication from the source identity to the destination identity.

I think we should emphasize the destination because `dports` is being checked on both `ingress` and `egress` maps of the L4Policy in the **destination** label.

List of Layer 4 port and protocol pairs which will be used in the **destination** of the communication from the source identity to the destination identity and vice versa.